vespacli/pyproject.toml

```diff
@@ -8,7 +8,7 @@ description = "A Python wrapper for Vespa CLI tools, supporting multiple platfor
 keywords = [ "vespa", "cli", "wrapper",]
 name = "vespacli"
 readme = "README.md"
-version = "8.dev"
+version = "8.0.dev"
```

Comment:
This is not correct, though? 🤔
If we are mapping the version with VespaCLI, 8.0.x is not correct, while 8.x it is.

Reply:
Changed back